source/common/common/key_value_store_base.cc

```diff
@@ -51,14 +60,34 @@ bool KeyValueStoreBase::parseContents(absl::string_view contents) {
   while (!contents.empty()) {
     absl::optional<absl::string_view> key = getToken(contents, error);
     absl::optional<absl::string_view> value;
+    absl::optional<std::chrono::seconds> ttl = absl::nullopt;
     if (key.has_value()) {
       value = getToken(contents, error);
     }
     if (!key.has_value() || !value.has_value()) {
       ENVOY_LOG(warn, error);
       return false;
     }
-    addOrUpdate(key.value(), value.value(), absl::nullopt);
+    if (checkForTtl(contents)) {
+      uint64_t ttl_int;
+      auto token = getToken(contents, error);
+      if (!token.has_value()) {
+        ENVOY_LOG(warn, error);
```

Comment:
Maybe prefix this error message with something? "Failed to read token" or something like that